cumulus/xcm/xcm-emulator/src/lib.rs

```diff
@@ -360,7 +361,14 @@ macro_rules! decl_test_relay_chains {
 			#[derive(Clone)]
 			pub struct $name;
 
+			use $crate::NetworkComponent;
+
 			impl $crate::Chain for $name {
+				Self::execute_with(|| {
+					let relay_block_number = <$name as NetworkComponent>::Network::relay_block_number();
+					$(<$on_initalize_fn>::on_initialize(relay_block_number);)?
+				})
+
```

Comment:
What is this doing in the impl?

Reply:
yeah I wasn't sure how to make these execute. I thought Self::execute could work. I'll remove